discord_slash/error.py

```diff
@@ -40,6 +40,20 @@ def __init__(self, name: str):
         super().__init__(f"Duplicate command name detected: {name}")
 
 
+class DuplicateCallback(SlashCommandError):
+    """
+    There is a duplicate component custom ID.
+    """
```

Comment:
```suggestion
    """Indicate exceptions where there is a duplicate component custom ID."""
```

Reply:
I don't whink it's consistent with other exceptions docstings, like:
-     Some formats are incorrect. See Discord API DOCS for proper format.
-     There is a duplicate command name.
-     Request to Discord API has failed.
-     There are duplicate :class:`.SlashCommand` instances.

Reply:
What about 
```suggestion
    """Raised when there is a duplicate component custom ID."""
```

Reply:
I'm not sure I follow, they're only docstrings in this case. Are we trying to strictly follow PEP-8 here?

Reply:
Yeah im gunna say this isnt worth delaying merge, feel free to discuss in staff chats. and possibly commit later